src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -668,11 +668,26 @@ public static void Initialize(Collection<Exception> errors, List<string> moduleP
             else
             {
                 // DSC SxS scenario
-                var psAssembly = Assembly.GetEntryAssembly();
-                var configSystemPath = Path.GetDirectoryName(psAssembly.Location);
+                //var psAssembly = Assembly.GetEntryAssembly();
+                var configSystemPath = Utils.DefaultPowerShellAppBase;
                 var systemResourceRoot = Path.Combine(configSystemPath, "Configuration");
-                var inboxModulePath = "Modules\\PSDesiredStateConfiguration";
-                if( !Directory.Exists(systemResourceRoot))
+                //var inboxModulePath = "Modules\\PSDesiredStateConfiguration";
+                ExecutionContext context = null;
+                CommandInfo commandInfo = new CmdletInfo("Get-Module", typeof(Microsoft.PowerShell.Commands.GetModuleCommand), null, null, context);
+                System.Management.Automation.Runspaces.Command getModuleCommand = new System.Management.Automation.Runspaces.Command(commandInfo);
+                var psDesiredStateConfigurationModule = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace)
+                .AddCommand(getModuleCommand)
```

Comment:
This is swallowing any possible indication of an error/warning. Do you really want to do this? It means a completely undiagnosable state if something goes wrong.

Reply:
Done. Sorry - intermittent commit.